be/src/storage/lake/rowset_update_state.cpp

```diff
@@ -406,43 +551,135 @@ Status RowsetUpdateState::_resolve_conflict(const TxnLogPB_OpWrite& op_write, co
     for (uint32_t segment_id = 0; segment_id < num_segments; segment_id++) {
         std::vector<uint64_t>& new_rss_rowids = new_rss_rowids_vec[segment_id];
 
-        uint32_t num_rows = new_rss_rowids.size();
-        std::vector<uint32_t> conflict_idxes;
-        std::vector<uint64_t> conflict_rowids;
-        DCHECK_EQ(num_rows, _partial_update_states[segment_id].src_rss_rowids.size());
-        for (size_t i = 0; i < new_rss_rowids.size(); ++i) {
-            uint64_t new_rss_rowid = new_rss_rowids[i];
-            uint32_t new_rssid = new_rss_rowid >> 32;
-            uint64_t rss_rowid = _partial_update_states[segment_id].src_rss_rowids[i];
-            uint32_t rssid = rss_rowid >> 32;
-
-            if (rssid != new_rssid) {
-                conflict_idxes.emplace_back(i);
-                conflict_rowids.emplace_back(new_rss_rowid);
+        // reslove normal partial update
+        if (!op_write.txn_meta().partial_update_column_ids().empty()) {
```

Comment:
We can add two method `_resolve_conflict_partial_update()` and `_resolve_conflict_auto_increment()` here to make this  `_resolve_conflict` not that long, easy for reading.

Reply:
done